benchmarks/cpp/benchmark.cpp

```diff
@@ -18,11 +18,13 @@
 using namespace sphericart;
 // shorthand for all-past-1 generic sph only
 template<typename DTYPE>
-inline void compute_generic(int n_samples, int l_max, DTYPE *prefactors, DTYPE *xyz, DTYPE *sph, DTYPE *dsph, DTYPE* buffers) {
+inline void compute_generic(int n_samples, int l_max, DTYPE *prefactors, DTYPE *xyz, DTYPE *sph, DTYPE *dsph, DTYPE* ddsph, DTYPE* buffers) {
     if (dsph==nullptr) {
-        generic_sph<DTYPE, false, false, 1>(xyz, sph, dsph, n_samples, l_max, prefactors, buffers);
+        generic_sph<DTYPE, false, false, false, 1>(xyz, sph, dsph, ddsph, n_samples, l_max, prefactors, buffers);
```

Comment:
I wouldn't bother because this is not part of the interface anymore... It's only in the benchmark file. Our interface now has three different functions:
`compute`
`compute_with_gradients`
`compute_with_hessians`
and they only take the arguments that are relevant, if I'm not mistaken